### PR TITLE
Remove '@fullscreen' from most tutorials

### PR DESCRIPTION
**Pull request by ganicke**
Drop the `@fullscreen` attribute from most (all except for 'RPS') tutorials.

RE: #2084

### CODE REVIEW

File: docs/projects/dice.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Let's turn the @boardname@ into a dice!
 
 ![A microbit dice](/static/mb/projects/dice.png)
 
-## Step 1 @fullscreen
+## Step 1
```
Comment (pelikhan) [inline docs/projects/dice.md:10]: I disagree the first step should pop out the hint.

File: docs/projects/flashing-heart.md [hunk 2]
```diff
@@ -8,13 +8,13 @@ Learn how to use the LEDs and make a flashing heart!
 
 ![Heart shape in the LEDs](/static/mb/projects/flashing-heart/sim.gif)
 
-## Step 1 @fullscreen
+## Step 1
```
Comment (pelikhan) [inline docs/projects/flashing-heart.md:11]: this is the very first tutorial, keep fullscreen on the first step

File: docs/projects/smiley-buttons.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ Code the buttons on the @boardname@ to show that it's happy or sad.
 
 ![Pressing the A and B buttons](/static/mb/projects/smiley-buttons/sim.gif)
 
-## Step 1 @fullscreen
+## Step 1
```
Comment (pelikhan) [inline docs/projects/smiley-buttons.md:10]: second tutorial, keep full screen here too.

File: docs/projects/spy/flashing-heart.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ Learn how to use the LEDs and make a flashing heart!
 
 ![Heart shape in the LEDs](/static/mb/projects/flashing-heart/sim.gif)
 
-## Step 1 @fullscreen
+## Step 1
```
Comment (pelikhan) [inline docs/projects/spy/flashing-heart.md:10]: keep full screen here it's the very first tutorial

**Comment by abchatra** [comment]
>> lgtm, though I'll also tag in @abchatra on if we want to do this now, or alongside the release

Thanks @jwunderl. I am ok merging the changes now as we want to do this anyway.

**Pull request merged**